app/services/booking_response.rb

```diff
@@ -1,9 +1,44 @@
 class BookingResponse
-  def initialize(success:)
-    @success = success
+  ALREADY_PROCESSED_ERROR = 'already_processed'.freeze
+  PROCESS_REQUIRED_ERROR = 'process_required'.freeze
+  NOMIS_VALIDATION_ERROR = 'nomis_validation_error'.freeze
+  NOMIS_API_ERROR = 'nomis_api_error'.freeze
+
+  attr_reader :error
+
+  def self.succesful
+    new(nil)
```

Comment:
Yes, this isn't clear I'll refactor it.

Regarding the pattern I think it's better wrapping the constants in class methods since it avoids getting unexpected values passed in and the actual values are an implementation detail